python_examples/luxembourg_example/luxembourg_repeat_sims_loc_cst_interventions.py

```diff
@@ -0,0 +1,73 @@
+#
```

Comment:
Is this the same as `python_examples/luxembourg_example/luxembourg_repeat_sims_loc_cst.py` but with the intervention parameters json? Again, just have one script, and in the comments explain how you have change json input to get both functionalities. This should also be documented in the readme

Reply:
Changed